src/app/rpmostree-builtin-status.c

```diff
@@ -563,6 +563,12 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
           print_kv ("Unlocked", max_key_len, unlocked);
           g_print ("%s%s", bold_suffix, red_suffix);
         }
+      const char *end_of_life_string = NULL;
+      /* look for endoflife attribute in the deployment */
+      g_variant_dict_lookup (dict, "endoflife", "&s", &end_of_life_string);
+
+      if (end_of_life_string)
+       g_print ("\nEndOfLife: %s", end_of_life_string);
```

Comment:
Actually, (sorry for the bikeshedding!), any reason we don't just print it like any other kv using `print_kv`? (Preferably in red & bold as well).